examples/grafana/useKubeAuth/jwt-example.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: grafana.integreatly.org/v1beta1
+kind: Grafana
+metadata:
+  name: jwt-grafana-ca
+spec:
+  # version: 12.2.0 # or newer
+  client:
+    useKubeAuth: true
+  disableAdminSecret: true
```

Comment:
Let's add a comment here stating that grafana-operator no longer needs to have the admin credentials for its API requests, so they can be disabled.